components/prism-clojure.js

```diff
@@ -0,0 +1,13 @@
+// Copied from https://github.com/jeluard/prism-clojure
+Prism.languages.clojure = {
+	comment: /;+.*/,
+	string: /"(?:\\.|[^\\"\r\n])*"/,
+	operator: /(::|[:|'])\b[a-zA-Z][a-zA-Z0-9*+!-_?]*\b/g, //used for symbols and keywords
```

Comment:
Can you make the group non-capturing here? And also remove the `g` flag. You could add the `i` flag to shorten the regexp a bit, and use `\w` in the second character class. Finally, I think the last `-` is supposed to be escaped (or moved at the end).
The whole pattern could be something like:
`/(?:::|[:|'])\b[a-z][\w*+!?-]*\b/i`

Reply:
Resolved